src/platform/export/exportToPythonPlain.ts

```diff
@@ -3,34 +3,47 @@
 
 'use strict';
 
+import { inject, injectable } from 'inversify';
 import { CancellationToken, NotebookCell, NotebookCellKind, NotebookDocument, Uri } from 'vscode';
 import { appendLineFeed } from '../../webviews/webview-side/common';
-import { IFileSystem } from '../common/platform/types';
+import { IFileSystem, IPlatformService } from '../common/platform/types';
 import { IConfigurationService } from '../common/types';
-import { IExport } from './types';
+import { ExportFormat, IExport, IExportDialog } from './types';
 
 // Handles exporting a NotebookDocument to python
-export class ExportToPythonPlainBase implements IExport {
+@injectable()
+export class ExportToPythonPlain implements IExport {
```

Comment:
I'm confused at why this class is the only Exporter that implements IExport

Reply:
I'm not sure either, it's a special case introduced via https://github.com/microsoft/vscode-jupyter/commit/621acee2bc6e5e00c7d398342e1a03f2f6bec04c

Reply:
I think it's just because it's the only export method that doesn't use nbConvert. Doesn't feel like a good reason to split the interface to me, but not something to worry about for this change.